.github/workflows/workflow.yaml

```diff
@@ -45,7 +45,7 @@ jobs:
     strategy:
       matrix:
         os: [ "ubuntu-latest", "macos-latest" ]
-        opa: [ "v0.42.2" ]
+        opa: [ "v0.54.0" ]
```

Comment:
Thanks for the contribution @anderseknert - Is opa `v0.54.0` required for this change?  We should just be aware of allowing future Rego changes that might not be compatible with previous Gatekeeper versions.  Currently we test with n-1, for which Gatekeeper v3.12 uses opa `v0.49.2`.

Reply:
Sure thing! I've just run the tests with v0.44.0 without issues, so I can set the version to that.

Will proceed with patching the versions. Thanks!

Reply:
Thanks - Those folders look good now!